packages/video_player/video_player_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.3.0
+
+* Update Pigeon to ^1.0.16.
```

Comment:
fyi: The style guide just says it should be the present tense, it doesn't specify which conjugation should be used (ie 3rd person singular, neutral).

Reply:
done

Reply:
It doesn't just say it should be in present tense, it says it should be "present tense indicative". I assumed that the fact the package was the implied subject was obvious, but I'll add that.

It also specifically says:
> For example, "Adds cool new feature.", not "Add" or "Added"

So I'm not clear how it's ambiguous that this should be "Updates" rather than "Update". But I'll add even more specific grammar notes there if you like.